### PR TITLE
Loyalty rewards

### PR DESCRIPTION
**Pull request by AndyGura**

### CODE REVIEW

File: src/auction/src/orders.mo [hunk 1]
```diff
@@ -191,9 +191,9 @@ module {
 
       if (not isPartial) {
         assetInfo.totalExecutedOrders += 1;
-        order.userInfoRef.loyaltyPoints += C.LOAYLTY_REWARD.ORDER_FULFILLMENT_APPENDIX;
       };
-      order.userInfoRef.loyaltyPoints += quoteVolume * C.LOAYLTY_REWARD.ORDER_FULFILLMENT_COEFFICIENT;
+      let loayltyVolumeReward = Float.fromInt(quoteVolume) * C.LOAYLTY_REWARD.ORDER_VOLUME |> Int.abs(Float.toInt(_));
```
Comment (timohanke) [inline src/auction/src/orders.mo:195]: Please fix typo "loaylty" to "loyalty" everywhere.

Comment (timohanke) [reply src/auction/src/orders.mo:195]: Let's do this without Floats:
```
let loyaltyVolumeReward = quoteVolume / 100_000;
```

**Review by timohanke — COMMENTED** [review]
Requested some changes

File: src/auction/src/orders.mo (hunk 1 above)
Comment (AndyGura) [reply src/auction/src/orders.mo]: Fixed

**Pull request merged**